features/bevragen/persoon/naam/dev/fields-gba.feature

```diff
@@ -45,3 +45,25 @@ Functionaliteit: GBA persoon: naam
     | naam.adellijkeTitelPredicaat.code         |
     | naam.adellijkeTitelPredicaat.omschrijving |
     | naam.adellijkeTitelPredicaat.soort        |
+
+  Scenario: 'volledige naam' wordt gevraagd met field pad 'naam.volledigeNaam'
+    Gegeven de persoon met burgerservicenummer '000000619' heeft de volgende gegevens
+    | geslachtsaanduiding (04.10) | voornamen (02.10) | adellijke titel of predicaat (02.20) | voorvoegsel (02.30) | geslachtsnaam (02.40) |
```

Comment:
Volgens fields-mapping-gba wordt naam.volledigeNaam vertaald naar naam|geslacht.

In naam zit ook aanduidingNaamgebruik. Die is op zich niet nodig voor volledigeNaam.

Dit scenario is nu impliciet over of aanduidingNaamgebruik ook meegeleverd wordt, omdat die niet is opgenomen in de Gegeven stappen.

aanduidingNaamgebruik hier toevoegen, of fields-mapping-gba.csv meer gedetailleerd beschrijven?

Reply:
allebei gedaan